tests/bluetooth/tester/src/btp.h

```diff
@@ -0,0 +1,102 @@
+/* bttester.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/util.h>
+#include <zephyr/bluetooth/addr.h>
+
+#define BTP_MTU 1024
+#define BTP_DATA_MAX_SIZE (BTP_MTU - sizeof(struct btp_hdr))
+
+#define BTP_INDEX_NONE		0xff
+
+#define BTP_SERVICE_ID_CORE	0
+#define BTP_SERVICE_ID_GAP	1
+#define BTP_SERVICE_ID_GATT	2
+#define BTP_SERVICE_ID_L2CAP	3
+#define BTP_SERVICE_ID_MESH	4
+
+#define BTP_STATUS_SUCCESS	0x00
+#define BTP_STATUS_FAILED	0x01
+#define BTP_STATUS_UNKNOWN_CMD	0x02
+#define BTP_STATUS_NOT_READY	0x03
+
+struct btp_hdr {
+	uint8_t  service;
+	uint8_t  opcode;
+	uint8_t  index;
+	uint16_t len;
+	uint8_t  data[];
+} __packed;
+
+#define BTP_STATUS			0x00
+struct btp_status {
+	uint8_t code;
+} __packed;
+
+/* Core Service */
+#define CORE_READ_SUPPORTED_COMMANDS	0x01
+struct core_read_supported_commands_rp {
+	uint8_t data[0];
+} __packed;
+
+#define CORE_READ_SUPPORTED_SERVICES	0x02
+struct core_read_supported_services_rp {
+	uint8_t data[0];
+} __packed;
+
+#define CORE_REGISTER_SERVICE		0x03
+struct core_register_service_cmd {
+	uint8_t id;
+} __packed;
+
+#define CORE_UNREGISTER_SERVICE		0x04
+struct core_unregister_service_cmd {
+	uint8_t id;
+} __packed;
+
+/* events */
+#define CORE_EV_IUT_READY		0x80
```

Comment:
I guess those should be moved to `btp_core.h` to match the BTP `txt` documentation file. I know that core service is implemented in `btp.c`, but I think it would be cleaner to have headers per service

Reply:
Done